### ISSUE TITLE
Move focus to router base on route change

### ISSUE THREAD

**Issue by developit** [preactjs/preact-cli#1169]
**Do you want to request a _feature_ or report a _bug_?**

Feature

**What is the current behaviour?**

No focus management for a11y by default.

**What is the expected behaviour?**

When navigating on the client, focus should be moved to the root of changed content (Router.base) to cause screen readers to announce the change. 

**If this is a feature request, what is motivation or use case for changing the behaviour?**

https://medium.com/@robdel12/single-page-apps-routers-are-broken-255daa310cf

https://twitter.com/robdel12/status/1258727047775600640?s=20
